cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -6,20 +6,21 @@
 
 #include <Camera/CameraComponent.h>
 #include <Components/InputComponent.h>
+#include <GameFramework/PlayerInput.h>
 
 #include "CoreUtils/Config.h"
 #include "CoreUtils/Unreal.h"
 #include "UrdfBot/UrdfParser.h"
 #include "UrdfBot/UrdfRobotComponent.h"
 
-AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn(object_initializer)
+AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer)
```

Comment:
I think this could still fit on a single line based on our new convention of 158 characters per line.

Reply:
fixed